docs/v1/P4Runtime-Spec.adoc

```diff
@@ -601,14 +601,18 @@ their write requests. gRPC provides authentication methods cite:[gRPCAuth] that
 should be deployed to prevent untrusted clients from creating channels, and thus
 from making changes or even reading the state of the server.
 
-=== Default Role
+=== Default Role and Default Device ID
 
 A controller can omit the role message in `MasterArbitrationUpdate`. This
 implies the "default role", which corresponds to "full pipeline access".
 This also implies that a default role has a `role_id` of `""` (default).
 If using a default role, all RPCs from the controller (e.g. `Write`) must
 leave the `role` unset.
 
+As per section [Default-Valued Fields](#sec-default-valued-fields), default values are invalid for all
+scalar P4 object IDs. This includes `device_id`, so the `0` ID must not be
+assigned to a device.
```

Comment:
Optional nit: maybe we can make this a bit more concise and precise by replacing: ", so the `0` ID must not be assigned to a device." -> "which must be nonzero."